### PR TITLE
chore: update main README and additional documentation

### PR DESCRIPTION
**Pull request by daniel-cit**
This PR updates the main README file and adds additional documentation

### CODE REVIEW

File: examples/standalone/README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # Standalone example
 
 This example:
-- Presents the deploy of the Secured Data Warehouse blueprint variant for ingesting encrypted data from on-prem sources.
+- Presents the deploy of the blueprint for ingesting encrypted data from on-premisses environments.
```
Comment (lanreogunmola) [inline examples/standalone/README.md:4]: Typo - Presents the deploy of the blueprint for ingesting encrypted data from on-premises environments

File: examples/standalone/README.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 - Uses the [Tink](https://developers.google.com/tink) library to encrypt data "on prem" that will be uploaded to BigQuery.
```
Comment (lanreogunmola) [inline examples/standalone/README.md:5]: Uploaded or streamed to BigQuery

File: examples/standalone/README.md [hunk 3]
```diff
@@ -192,7 +187,7 @@ Install the following dependencies:
 
 ### Cloud SDK configurations
 
-The standalone example uses `tinkey` to generate the wrapped_key that will be used to encrypt "on-prem" data. The `tinkey` tool runs using the [Application Default Credentials](https://cloud.google.com/sdk/gcloud/reference/auth/application-default).
+The standalone example uses `tinkey` to generate the wrapped_key that will be used to encrypt "on-premisses" data. The `tinkey` tool runs using the [Application Default Credentials](https://cloud.google.com/sdk/gcloud/reference/auth/application-default).
```
Comment (lanreogunmola) [inline examples/standalone/README.md:190]: typo -  on-premises

**Review by lanreogunmola — COMMENTED** [review]
Added a few edits and typos corrections

File: examples/standalone/README.md (hunk 1 above)
Comment (daniel-cit) [reply examples/standalone/README.md]: fixed

File: examples/standalone/README.md (hunk 2 above)
Comment (daniel-cit) [reply examples/standalone/README.md]: fixed

File: examples/standalone/README.md (hunk 3 above)
Comment (daniel-cit) [reply examples/standalone/README.md]: fixed

**Review by lanreogunmola — APPROVED** [review]

**Pull request merged**